Core/Core/Login/LoginFindSchoolViewController.swift

```diff
@@ -156,6 +157,11 @@ class LoginFindSchoolViewController: UIViewController {
             navigationItem.rightBarButtonItem = nil
         }
     }
+
+    func saveAccount(_ account: APIAccountResult?) {
```

Comment:
It seems that we only use the `domain` field of this `APIAccountResult`. Would it be possible to save this `String` instead of the whole object?

Reply:
If I remember correctly, the authentication method was also useful, but it's not used in this solution, is there a reason for that?

Reply:
also, the "name" field is used

Reply:
I used the object on purpose.

Reply:
What was that purpose?

Reply:
To access several variable. We use name and domain.

Reply:
Ok, I missed the case when an account is selected from the previous logins list.